ConsoleApp1/Library.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+  
+    static class Library
```

Comment:
Why did you mark class as static ?

Reply:
Well, at first I didn't have an intention to turn it into static. However when I created a few other classes I considered that static Library would be not a bad idea, especially in our case when we actually need 1 library.  Yes, I undestand that we should take into accont the neccessity to create more such libraries in future, but I thought it would be also a good practice to work with static class. And, indeed, the static library class helped me to understand better how static class works with non-static ones. That was my point :)